### ISSUE TITLE
Splash text on login page hides behind other elements

### ISSUE THREAD

**Issue by dsnellgrove** [crits/crits#373]
The splash text on login page hides behind other elements

Before e3e70432febe6cfd024d666d9cb3af2cc88ca8a1:
![before](https://cloud.githubusercontent.com/assets/3044831/5764380/8be92212-9cbf-11e4-918d-eb758f25ec4e.png)

After e3e70432febe6cfd024d666d9cb3af2cc88ca8a1:
![after](https://cloud.githubusercontent.com/assets/3044831/5764383/928ae858-9cbf-11e4-823a-452fa78bc83f.png)

**Comment by mgoffin** [comment]
This was an attempt to fix the issue prior to e3e7043 where the login page was horrifically uncentered (especially when TOTP isn't enabled).

It could probably use some tweaking as I think there's probably too many floating elements going on right now.

**Comment by dsnellgrove** [comment]
Yup. Fixing what I broke in the layout in my attempts to fix what was broke in the layout while making TOTP autohide. Time for more fixing what broke in the layout. Is this where we break down and use a library that does the layout for us like the Bootstrap grid?

**Comment by dsnellgrove** [comment]
(or the dozen+ jquery layout libraries now that I looked)

**Comment by mgoffin** [comment]
I would be fine with Bootstrap to be honest. Comes with a lot of great stuffs, and there's a lot of great plugins for it.
